src/content/docs/fr/2/guide/notification.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Notifications
+description: Envoie une notification native à l'utilisateur.
+i18nReady: true
+---
+
+import PluginBreadcrumb from '@components/PluginBreadcrumb.astro';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+import Stub from '@components/Stub.astro';
+
+<PluginBreadcrumb
+	version="v2.0.0-alpha.2"
+	github="https://github.com/tauri-apps/plugins-workspace/tree/v2/plugins/notification"
+	npm="https://www.npmjs.com/package/@tauri-apps/plugin-notification"
+	cratesio="https://crates.io/crates/tauri-plugin-notification"
+	jsApi="/2/reference/js/notification"

```

Comment:
```suggestion
	jsApi="/fr/2/reference/js/notification"

```

Reply:
Oh I thought the links would be updated later, thanks you for fixing all the links 👍